### DISCUSSION TITLE
Disable new user wizard and 'trusted' warning?

### DISCUSSION THREAD

**Discussion by scarolan** [coder/code-server#5511] [Q&A] [answered]
My use case is a bit unique - I'm using code server in a browser-based demo environment. 

I'd like to skip the "Do you trust this folder" warning and also the setup wizard that shows on a fresh installation.

Anyone know if there are startup or config flags I can set to skip these or auto-trust the directory?

**Comment by jsjoeio** [comment, marked as answer]
I think you can use these in your `settings.json`:

```json
"workbench.startupEditor": "none",
"security.workspace.trust.enabled": false,
```

**Reply by scarolan** [reply]
@jsjoeio you are my hero, thanks
